lib/wallet.js

```diff
@@ -21,6 +21,10 @@ function computeSeed (masterSeed) {
   return hkdf(masterSeed, 32, { salt: 'lamassu-server-salt', info: 'wallet-seed' })
 }
 
+function computeOperatorId (masterSeed) {
+  return hkdf(masterSeed, 16, { salt: 'lamassu-server-salt', info: 'operator-id' }).toString('hex')
+}
+
```

Comment:
We should have a module for this and use everywhere it applies. Also, this won't really work for pazuz, we need to get operatorID from asyncLocalStorage, no?

Reply:
To be done after some merges.